diff -ura /src/client/folder-list/folder-list-tree.vala /src/client/folder-list/folder-list-tree.vala
--- /src/client/folder-list/folder-list-tree.vala	2025-06-15 10:53:03.204022864 +0100
+++ /src/client/folder-list/folder-list-tree.vala	2025-06-15 11:05:40.720890763 +0100
@@ -49,8 +49,8 @@
     }
 
     public override void get_preferred_width(out int minimum_size, out int natural_size) {
-        minimum_size = 360;
-        natural_size = 500;
+        minimum_size = 215;
+        natural_size = 415;
     }
 
     public void set_has_new(Geary.Folder folder, bool has_new) {
